activerecord/test/cases/sanitize_test.rb

```diff
@@ -25,6 +25,15 @@ def test_sanitize_sql_array_handles_bind_variables
     assert_equal "name=#{quoted_bambi_and_thumper}", Binary.send(:sanitize_sql_array, ["name=?", "Bambi\nand\nThumper".mb_chars])
   end
 
+  def test_sanitize_sql_array_handles_named_bind_variables
+    quoted_bambi = ActiveRecord::Base.connection.quote("Bambi")
+    assert_equal "name=#{quoted_bambi}", Binary.send(:sanitize_sql_array, ["name=:name", name: "Bambi"])
+    assert_equal "name=#{quoted_bambi}", Binary.send(:sanitize_sql_array, ["name=:name", name: "Bambi".mb_chars])
```

Comment:
Why do we care about `mb_chars` here? Seems like that has nothing to do with testing that named bind variables are handled.